Source/EasyNetQ/RequestConfiguration.cs

```diff
@@ -10,18 +10,22 @@ namespace EasyNetQ
     /// </summary>
     public interface IRequestConfiguration
     {
+        /// <summary>
+        /// Sets a priority of the message
+        /// </summary>
+        /// <param name="priority">The priority to set</param>
+        IRequestConfiguration WithPriority(byte priority);
```

Comment:
this method is redeclared in several interfaces, maybe create an interface called IPublishConfiguration and inherit IFuturePublishConfiguration, ISendConfiguration and IRequestConfiguration from that?

Reply:
We already have `IPublishConfiguration` 🤣 Another name is needed

Also, interfaces of these operations could differ: for example, ISendConfiguration shouldn't have `WithTopic`(I mean a way to set routing key) because a queue is routing key.

There is one more thing: probably it's worth to create such a configuration for `AdvancedBus.Publish` method(probably it could be IAdvancePublishConfiguration, sorry), because I want to introduce an option `WithPublisherConfirms` which will allow enabling publisher confirms per request.